purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -521,18 +684,48 @@ class Purchases internal constructor(
      *
      * @param [activity] Current activity
      * @param [packageToPurchase] The Package you wish to purchase
+     * @param [isPersonalizedPrice] Set this parameter to true if your app can be distributed to users in the EU and
+     * the price you are offering is personalized. Ignored for Amazon.
+     * See https://developer.android.com/google/play/billing/integrate#personalized-price for more info.
      * @param [listener] The listener that will be called when purchase completes.
      */
     fun purchasePackage(
         activity: Activity,
         packageToPurchase: Package,
+        isPersonalizedPrice: Boolean,
         listener: PurchaseCallback
     ) {
         startPurchase(
             activity,
             // TODOBC5 Move this logic to StoreProduct
             packageToPurchase.product.defaultOption?.purchasingData ?: packageToPurchase.product.purchasingData,
             packageToPurchase.offering,
+            isPersonalizedPrice, // TODOC5 figure out isPersonalizedPrice with package
```

Comment:
Hmm can you explain this TODO? Also this should be `TODOBC5`

Reply:
oh woops i should clean this up anyway. was basically wondering if isPersonalizedPrice should be applied to a subscription option only, and didn't know how we would handle that on a package or product purchase, where the default is applied.

but we've since decided that we don't want to make the assumption that this only applies to offers..